extractors/building_parser.py

```
@@ -42,6 +51,37 @@ def pixmap_to_bgr(self, pix) -> np.ndarray:
+                
+                # 이미지 기반 PDF인 경우 OCR 수행
+                pix = page.get_pixmap(matrix=fitz.Matrix(1, 1))  # 낮은 해상도로 빠르게 확인
+                img = self.pixmap_to_bgr(pix)
+                
+                try:
+                    words = self.ocr_google_vision(img)
+                    # OCR 결과에서 "건축물대장" 찾기
+                    for word in words:
+                        if "건축물대장" in word[4]:
+                            return True
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)